package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "carbon-react",
-  "version": "97.0.0",
+  "version": "96.1.0",
```

Comment:
You can't change this manually, this is handled by our semantic release process

Reply:
Yes I know.... I have just noticed that this has automatically changed